packages/uploadthing/src/upload-builder.test.ts

```diff
@@ -62,7 +62,7 @@ it("passes `Request` by default", () => {
   const f = createBuilder();
 
   f(["image"]).middleware((req) => {
-    expectTypeOf(req).toMatchTypeOf<Request>();
+    expectTypeOf(req).toMatchTypeOf<Partial<Request> & RequestBase>();
```

Comment:
I don't like if this is exposed to the user - they should just see a `Request` / `NextRequest` or `NextApiRequest` etc